spec/2a_features.adoc

```diff
@@ -300,24 +322,12 @@ Feature data models <<B23>> from non-GeoPackage implementations that have multip
 The declared SQL type of the geometry column in a vector feature user data table SHALL be specified by the `geometry_type_name` column for that `column_name` and `table_name` in the `gpkg_geometry_columns` table.
 ====
 
-[[r150]]
-[caption=""]
-.Requirement 150
-====
-A feature table or view SHALL have a column that uniquely identifies the row. For a feature table, the column SHOULD be an integer primary key. If there is no primary key column, the first column SHALL be of type INTEGER and SHALL contain unique values for each row.
-====
-
-[NOTE]
-=====
-Using an integer primary key in a feature table allows features to be linked to row level metadata records in the `gpkg_metadata` table by rowid <<B5>> values in the `gpkg_metadata_reference` table as described in clause <<_metadata_reference_table>> below. Since the concept of primary keys does not exist for views in SQLite, this requirement also provides a way to produce a compliant feature view with a discoverable key-like column^<<K17>>^.
-=====
-
 [#example_feature_table_cols,reftext='{table-caption} {counter:table-num}']
 .EXAMPLE : Sample Feature Table or View^<<K17>>^ Definition
 [cols=",,,,,",options="header"]
 |=======================================================================
 |Column Name |Type |Description |Null |Default |Key
-|`id` |INTEGER |Autoincrement^<<K6a>>^ primary key |no | |PK
+|`id` |INTEGER |Autoincrement^<<K6a>>^ primary key |N/A | |PK
```

Comment:
Why does `Null` change from "no" to "N/A" here?

Reply:
Because SQLite ignores the Null flag for primary key columns that are backed by a rowid. SQLite will populate them either way.

Reply:
Ack.

Reply:
I agree with the `N/A` notation. However, I think useful to implementers to create additionally a K6b note paraphrasing @jyutzler answer to provide them context.